packages/nuxt/src/components/module.ts

```diff
@@ -253,6 +253,7 @@ export default defineNuxtModule<ComponentsOptions>({
             config.plugins.push(ComponentsChunkPlugin.vite({
               getComponents,
               buildDir: nuxt.options.buildDir,
+              isRolldownCompatEnabled: nuxt.options.experimental.rolldownCompat,
```

Comment:
is it possible to detect this without requiring the user to set a flag?

Reply:
I tried before (see PR description [and this commit](https://github.com/nuxt/nuxt/pull/31812/commits/4c95619eb5cedacf4677ec2130ff50a2a0e62ebf#diff-87425e8bfe7538c62da5fe7af431df0b50a5e46704cfcdb6fb525c9607ba03caR201)) but this would've led to implicitly bundling vite. If we can avoid that (externalizing?) I think it'd be possible.